src/backend/opencl/kernel/orb.hpp

```diff
@@ -29,8 +29,24 @@ using cl::LocalSpaceArg;
 using cl::NDRange;
 using std::vector;
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
+#if defined(__clang__)
```

Comment:
Clang will accept GCC pragmas but not the other way around. You should remove the clang pragmas.